src/FileList/FileExtInfoLoader.cc

```diff
@@ -107,6 +107,10 @@ bool FileExtInfoLoader::FillFileExtInfo(
     bool info_ok(false);
     if (_loader && _loader->CanOpenFile(message)) {
```

Comment:
_loader is a nullptr for an unsupported file or image type.  Wouldn't it be better to catch this in Session, before it is passed to the FileExtInfoLoader constructor and FillFileExtInfo() is called?

The message should probably use the word "file" instead of "image" since it could be, as in the subject of this PR, a non-image file.  It could also be an unsupported image type, a file permissions issue, or other problems. @kswang1029  how can this message cover these different scenarios and not be confusing?  Or am I overthinking this?

Reply:
Good point. I think we simply say “the file cannot be loaded”. If we, however, can know _why_ it cannot be loaded _without extra work_, we can also show the reasons (unsupported format, permission, broken file etc) to users.

Reply:
if there is an equivalent handy “file” command that we can use, it may be good to use it and return the output as part of our error message.

Reply:
Agree, can check if file does not exist else if file is not readable else if !_loader, with a message for each.  We have to do extra checking since this is user input.

Reply:
I think the backend already checks if the file exists or is readable @kswang1029 could you please confirm it?

Reply:
If it is permission issue, we will see a popup error message on the frontend. 
![Screen Shot 2021-09-27 at 12 55 38 PM](https://user-images.githubusercontent.com/20819712/134847506-bd7ed567-5424-42dc-a1d0-d7591fae7395.png)

If it is a file that does not exist, the frontend loads without any error/warning messages. The file browser shows up just like usual.

Reply:
@veggiesaurus @kswang1029 is this resolved?  The permissions error message is coming from casacore::RegularFileIO and could be improved.  And according to the previous comment, if the file does not exist it is ignored.  Is there a message in the log/console?  If I were a user I would be confused.

Reply:
I think if we load a file that does not exist, we should show an error message like "Input file does not exist.".

Reply:
The `carta_backend` can print an error message "File does not exist." if we load a file that does not exist. But I am not sure does the `carta` wrapper can catch this message? Or the `carta_backend` should throw an exception with the message "File does not exist." to let the `carta` catch this error message?

Reply:
I think we should let `carta_backend` throw the error message. One possible scenario is a user uses the GUI to load an image but after receiving the file list, the selected image is deleted somehow. So in this case, when the `carta_backend ` loads the image, it can warn the user.

Reply:
@kswang1029 With current code changes, can the `carta` show the error message `Unsupported format.` if we load a non-image file? If not, this error message should also come up with an exception thrown.

Reply:
Hmm I am not sure if I can test the wrapper properly. I do see unsupported image error on the frontend.

Reply:
In my test, the frontend can show `File does not exist` message if we load a file that does not exist. Perhaps we can verify it after the new `carta` is ready.

Reply:
Ok